expirationd.lua

```diff
@@ -347,6 +362,23 @@ local function expirationd_run_task(name, space_id, is_tuple_expired, options)
     end
     task.index = expire_index
 
+    -- check iterator_type
+    if options.iterator_type ~= nil then
+        task.iterator_type = options.iterator_type
+    end
+
+    -- check start_key
+    if options.start_key ~= nil then
```

Comment:
nit: it disallows start_key to be `box.NULL`.

And again. There are no any tests that cover your changes.

Reply:
Fixed